fw/http_parser.c

```diff
@@ -6308,6 +6308,15 @@ __h2_req_parse_content_length(TfwHttpMsg *msg, unsigned char *data, size_t len,
 			      bool fin)
 {
 	int ret;
+	TfwHttpReq *req = (TfwHttpReq *)msg;
+
+	if (unlikely(req->method == TFW_HTTP_METH_GET
```

Comment:
Can't method override occur?

Reply:
That's possible. We should take it into account.